README.md

```diff
@@ -12,6 +12,7 @@ Inputs:
 * `repository` The GitHub repository, defaults to the `GITHUB_REPOSITORY` env var
 * `branch` The branch to publish charts, defaults to `gh-pages`
 * `helm_version` The Helm CLI version, defaults to the latest release
+* `target_dir` The target directory to store the charts, defaults to `.`
```

Comment:
Can you please move `target_dir ` right after `branch`.

Reply:
Done